pkg/load/agents/utils.go

```diff
@@ -14,14 +14,17 @@ import (
 	"k8s.io/test-infra/prow/interrupts"
 )
 
-func startWatchers(path string, callback func() error, metric *prometheus.CounterVec, universalSymlinkWatcher *UniversalSymlinkWatcher) error {
+func startWatchers(path string, errCh chan<- error, callback func() error, metric *prometheus.CounterVec, universalSymlinkWatcher *UniversalSymlinkWatcher) error {
 	var watchers []func(context.Context)
 
 	if universalSymlinkWatcher != nil {
 		watcher := func(ctx context.Context) {
 			errFunc := func(err error, msg string) {
 				recordErrorForMetric(metric, msg)
 				logrus.WithError(err).Error(msg)
+				go func() {
+					errCh <- err
+				}()
```

Comment:
We'll also need to propagate errors from the other watchers (the `else` branch).  But it's fine if we implement just the universal watcher part in this pull request.

For now, we can just pass `nil` as the channel parameter everywhere we don't use this implementation, instead of creating unused channels.